manager/controllers/advisories.go

```diff
@@ -41,12 +42,39 @@ type AdvisoryInlineItem struct {
 	AdvisoryItemAttributes
 }
 
+type AdvisoriesSums struct {
+	Total       int64 `query:"count(*)" gorm:"column:total"`
+	Unknown     int64 `query:"count(*) filter (where am.advisory_type_id = 0)" gorm:"column:unknown"`
+	Unspecified int64 `query:"count(*) filter (where am.advisory_type_id = 1)" gorm:"column:unspecified"`
+	Other       int64 `query:"count(*) filter (where am.advisory_type_id = 2)" gorm:"column:other"`
+	Enhancement int64 `query:"count(*) filter (where am.advisory_type_id = 3)" gorm:"column:enhancement"`
+	Bugfix      int64 `query:"count(*) filter (where am.advisory_type_id = 4)" gorm:"column:bugfix"`
+	Security    int64 `query:"count(*) filter (where am.advisory_type_id = 5)" gorm:"column:security"`
```

Comment:
Nice. This is correct except for Unknown, Unspecified and  Other :).
The thing is we care only about Security, Bugfix, Enhancement and anything else is counted under Other.
See e.g. metrics
https://github.com/RedHatInsights/patchman-engine/blob/master/vmaas_sync/metrics.go#L256